### DISCUSSION TITLE
Recommendation for hosting closed source app

### DISCUSSION THREAD

**Discussion by Rik44444** [ImranR98/Obtainium#1152] [Q&A] [answered]
Thanks for your development of this great app. 
Do you have a recommendation for where to post closed-source / proprietary apps so that they can be most easily retrieved by your system?

**Comment by mhbcrypto** [comment, marked as answer]
They're on APKMirror and APKPure, and you can set Obtainium to watch each one for APK updates.

**Comment by ImranR98** [comment]
Yes, one of the existing sources like someone suggested. I'd recommend just putting up your own webpage with links to APK files.
